docs/info/privacy-policy.md

```diff
@@ -0,0 +1,369 @@
+---
+template: main.html
+description: Privacy policy
+hide:
+
+- navigation
+
+---
+
+# Privacy Policy
+
+Effective date: 2022-12-01
+
+## 1. Introduction
+
+ExpressLRS (“us”, “we”, or “our”) operates https://www.expresslrs.org (hereinafter referred to as “Service”).
+
+Our Privacy Policy governs your visit to https://www.expresslrs.org, and explains how we collect, safeguard and disclose
+information that results from your use of our Service.
+
+We use your data to provide and improve Service. By using Service, you agree to the collection and use of information in
+accordance with this policy.
+
+## 2. Definitions
+
+SERVICE means the https://www.expresslrs.org website operated by ExpressLRS.
+
+USAGE DATA is data collected automatically either generated by the use of Service or from Service infrastructure
+itself (for example, the duration of a page visit).
+
+COOKIES are small files stored on your device (computer or mobile device).
+
+DATA CONTROLLER means a natural or legal person who (either alone or jointly or in common with other persons) determines
+the purposes for which and the manner in which any personal data are, or are to be, processed. For the purpose of this
+Privacy Policy, we are a Data Controller of your data.
+
+DATA PROCESSORS (OR SERVICE PROVIDERS) means any natural or legal person who processes the data on behalf of the Data
+Controller. We may use the services of various Service Providers in order to process your data more effectively.
+
+DATA SUBJECT is any living individual who is the subject of Personal Data.
+
+THE USER is the individual using our Service. The User corresponds to the Data Subject, who is the subject of Personal
+Data.
+
+## 3. Information Collection and Use
+
+We collect several different types of information for various purposes to provide and improve our Service to you.
+
+## 4. Types of Data Collected
+
+While using our Service, we may ask you to provide us with certain personally identifiable information that can be used
+to contact or identify you (“Personal Data”). Personally identifiable information may include, but is not limited to:
+
+0.1. Email address
```

Comment:
I don't think these sections apply on our website.

We don't collect these in any shape or form. 
we can leave just the Usage Data, but I don't think personal info are collected.

Reply:
This is just future proofing. For example, if we decide to add newsletter subscription form this section would be useful.